src/main/java/com/microtripit/mandrillapp/lutung/Main.java

```diff
@@ -0,0 +1,30 @@
+package com.microtripit.mandrillapp.lutung;
+
+import com.microtripit.mandrillapp.lutung.controller.MandrillUsersApi;
+import com.microtripit.mandrillapp.lutung.model.MandrillApiError;
+import org.springframework.beans.factory.config.PropertyPlaceholderConfigurer;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import java.io.IOException;
+import java.util.Properties;
+
+/**
+ * Created by yotam on 12/26/15.
+ */
+public class Main {
+    public static void main(String[] args) throws IOException, MandrillApiError {
+        final ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext("classpath:applicationContext-LutungMandrill-all.xml");
```

Comment:
I kind of disagree adding a Main class to the repo to show how to "wire" it all up together. I think if you want to have an example, you can better have this as a part of the README.